cryk-ui/src/pages/Cryptocurrencies.tsx

```diff
@@ -55,6 +61,12 @@ export default function Cryptocurrencies() {
         )
     }
 
+    const handleChange = (pageNumber: number) => {
+        setCurrentPage(pageNumber);
+    };
+
+    const count =  Math.ceil(data.cryptocurrenciesInfo.totalCount / cryptosPerPage);
```

Comment:
I think we'll need a state for that variable and we'll update it within `useEffect`:
```javascript   
 useEffect(() => {
        if (data) {
            setCryptocurrencies(data.cryptocurrencies);
            setTotalNumberOfCryptocurrencies(Math.ceil(data.cryptocurrenciesInfo.totalCount / CRYPTOS_PER_PAGE));
        }
    }, [loading, data]);
```